openfisca_france/model/prestations/mobilite_jeunes.py

```diff
@@ -0,0 +1,68 @@
+from openfisca_france.model.base import Individu, Variable, MONTH
+from openfisca_france.model.prestations.education import TypesClasse
+
+
+class aide_mobilite_parcoursup_sortie_academie(Variable):
+    value_type = bool
+    label = "Indicatrice d'une sortie de l'académie d'études au lycée dans les voeux ParcourSup"
+    entity = Individu
+    definition_period = MONTH
```

Comment:
Pourquoi cette variable pourrait-elle varier de mois en mois ? 🤔 

```suggestion
    definition_period = ETERNITY
```